### ISSUE TITLE
Support global packages

### ISSUE THREAD

**Issue by vicentedealencar** [th0r/npm-upgrade#8]
First of all thanks for this tool, its is a real time saver.

Have you considered upgrading packages installed globally with `-g` flag?

**Comment by tunnckoCore** [comment]
That's good idea too, definitely!

**Comment by th0r** [comment]
Good idea, guys, but unfortunately I don't have enough time to implement it. So PR is welcome.

**Comment by Medallyon** [comment]
I implemented this in #70. Could you have a look at it? 😺

**Comment by th0r** [comment]
Released in 3.1.0
